lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -225,7 +225,7 @@ def self.ruby_method_runnable?(aem)
 
     def self.setup_drb_for_ruby_method
       require 'drb/timeridconv'
-      DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
+      @@global_id_conv = DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
       drb_front  = MiqAeMethodService::MiqAeServiceFront.new
       drb        = DRb.start_service("druby://127.0.0.1:0", drb_front)
```

Comment:
@mkanoor I'm fine with this approach for now.  Note, I don't think you need a class variable, `@@global_id_conv` .  Using `@global_id_conv` should be enough since we're in the context of a class method.